opal/core/search/extract.py

```diff
@@ -107,6 +149,11 @@ def zip_archive(episodes, description, user):
         make_file_path = functools.partial(os.path.join, target_dir, zipfolder)
         zip_relative_file_path = functools.partial(os.path.join, zipfolder)
 
+        file_name = "data_dict.html"
```

Comment:
Could this become `data_dictionary.html`  ?
(I'm not sure dictionary -> dict is a common contraction outside of Python programmers :) )